metacat-connector-polaris/src/main/java/com/netflix/metacat/connector/polaris/store/PolarisConnector.java

```diff
@@ -15,7 +19,7 @@
  * This class exposes APIs for CRUD operations.
  */
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
-public class PolarisConnector {
+public class PolarisConnector implements PolarisStoreService {
```

Comment:
Should we rename `PolarisConnector` to something like `PolarisStoreConnector`? Since we already have other connector classes with a similar name.

Reply:
sure.